core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -6,7 +6,7 @@
     <i class="material-icons oppia-navbar-close-icon" ng-if="$ctrl.isSidebarShown()" >&#10005;</i>
   </a>
 
-  <a class="oppia-navbar-brand-name oppia-transition-200 protractor-test-oppia-main-logo d-none d-sm-block" href="/"
+  <a class="oppia-navbar-brand-name oppia-transition-200 protractor-test-oppia-main-logo d-none d-sm-block" href="/splash"
```

Comment:
Same thing. Please fix all of these.

Reply:
Addressed [above](https://github.com/oppia/oppia/pull/9109/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js&file-filters%5B%5D=.py&file-filters%5B%5D=.ts#r415421846).